test/test_pipeline/components/setup/test_setup_networks.py

```diff
@@ -47,8 +47,6 @@ def test_pipeline_fit(self, fit_dictionary_tabular, embedding, backbone, head):
         assert backbone == config.get('network_backbone:__choice__', None)
         assert head == config.get('network_head:__choice__', None)
         pipeline.set_hyperparameters(config)
-        # Need more epochs to make sure validation performance is met
-        fit_dictionary_tabular['epochs'] = 100
```

Comment:
Why do you want to change this?

Reply:
because now the fit dictionary already has 100 for the epochs value. See this https://github.com/ravinkohli/Auto-PyTorch/blob/ad43a947a5447c75b4ccd6bd44335fb96aa226d8/test/conftest.py#L290